packages/react-data-grid/src/common/utils/index.ts

```diff
@@ -0,0 +1,51 @@
+// eslint-disable-next-line
+declare var Immutable: any;
```

Comment:
I think we should just import Immutable and not rely on `window.Immutable`.
This file might need an overhaul, don't think we really need this `isFunction`.

Reply:
That would be better and we can add `ImmutableJS` as a peer dependency. I also wonder if we even need to support immutable js or not.